bundle/edu.gemini.ags/src/main/java/edu/gemini/ags/gems/GemsGuideStarSearchOptions.java

```diff
@@ -23,36 +22,28 @@
  *
  * See OT-25
  */
-public class GemsGuideStarSearchOptions {
+public final class GemsGuideStarSearchOptions {
 
     public enum CatalogChoice {
-        PPMXL_GEMINI("PPMXL@Gemini", "PPMXL at Gemini"),
-        UCAC4_GEMINI("UCAC4", "UCAC4 at Gemini"),
+        PPMXL_GEMINI(PPMXL$.MODULE$, "PPMXL at Gemini"),
+        UCAC4_GEMINI(UCAC4$.MODULE$, "UCAC4 at Gemini"),
         ;
```

Comment:
`CatalogChoice` is defined here but not really used except in the GUI. Instead we fix the catalog to `PPMXL` in `GemsStrategy` and `Pwfs1SouthNGS2`. In the GUI if the user selects `UCAC4` it is used to find the tip-tilt correction Canopus options, but not to find the PWFS1 options.  Somehow I believe the catalog has to be configurable in both contexts (AGS and manual/GUI) and the same across both guide star types.